README.md

```diff
@@ -88,10 +89,18 @@ let agentConfig : AgentConfiguration = {
   sendConsoleEvents: true,
 
   // Optional: Enable or disable reporting data using different endpoints for US government clients.
-   fedRampEnabled: false
+   fedRampEnabled: false,
 
   // Optional: Enable or disable offline data storage when no internet connection is available.
-  offlineStorageEnabled:true
+  offlineStorageEnabled:true,
```

Comment:
Still just want to make sure the default is false for offlineStorageEnabled, backgroundReportingEnabled, and newEventSystemEnabled.

Reply:
i enabled it for all the hybrid agents.

Reply:
Offline storage should be disabled by default in all agents